### PR TITLE
Fixed 19573: Alignment Issue when create a new email template admin

### PR DESCRIPTION
**Pull request by suryakant-krish**
### Description (*)

Alignment Issue when create a new email template admin.
### Fixed Issues (if relevant)

1. magento/magento2#19573: Alignment Issue when create a new email template admin.

### Manual testing scenarios (*)

1. Login into admin panel > Marketing > Email Templates > Create new Email template Load any default template. Current used For Alignment issue.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @suryakant-krish. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by suryakant-krish** [comment]
I have applied a solution for a specific email template section because when I add margin it will affect on other section. 

Here is my commit id : 72438de

File: app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less [hunk 1]
```diff
@@ -147,6 +147,14 @@
                 }
             }
         }
+
+        &#currently_used_for {
```
Comment (sivaschenko) [inline app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less]: Please avoid using id's in css selectors, try to build css selectors based on classes

**Review by sivaschenko — CHANGES_REQUESTED** [review]
@suryakant-krish thanks for the contribution, please see my code review notes.

File: app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less (hunk 1 above)
Comment (suryakant-krish) [reply app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less]: I have updated code using class.

**Comment by sivaschenko** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, here is your [new Magento instance](https://pr-19574.instances.magento-community.engineering).
Admin access: [https://pr-19574.instances.magento-community.engineering/admin](https://pr-19574.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`

**Comment by sivaschenko** [comment]
Actual result:
image

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-3627](https://jira.corp.magento.com/browse/ENGCOM-3627) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @suryakant-krish. Thank you for your contribution. 
We will aim to release these changes as part of 2.3.1. 
Please check the release notes for final confirmation.

**Pull request merged**